From 9664825818ce2c126ebbbc92adcd5fe04bccb31b Mon Sep 17 00:00:00 2001
From: 5schatten <supervisedthinking@gmail.com>
Date: Wed, 16 Oct 2019 17:04:57 +0200
Subject: [PATCH] Makefile: define MESA_EGL_NO_X11_HEADERS for eglplatform.h if
 mesa is used

---
 Makefile | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/Makefile b/Makefile
index 4902885e..fff15450 100644
--- a/Makefile
+++ b/Makefile
@@ -138,6 +138,7 @@ else ifneq (,$(findstring rpi,$(platform)))
    endif
    ifeq ($(MESA), 1)
       GL_LIB := -lGLESv2
+      COREFLAGS += -DMESA_EGL_NO_X11_HEADERS
    else
       LLE = 0
       CPUFLAGS += -DVC
@@ -250,6 +251,10 @@ else ifneq (,$(findstring AMLG,$(platform)))
       endif
    endif
 
+   ifneq (,$(findstring mesa,$(platform)))
+     COREFLAGS += -DMESA_EGL_NO_X11_HEADERS
+   endif
+
    GL_LIB := -lGLESv2
    HAVE_NEON = 1
    WITH_DYNAREC=arm
